firebase.json

```diff
@@ -153,6 +153,7 @@
       { "source": "/go/dart2js-info", "destination": "https://github.com/dart-lang/sdk/tree/main/pkg/dart2js_info", "type": 301 },
       { "source": "/go/dartdoc-options-file", "destination": "https://github.com/dart-lang/dartdoc#dartdoc_optionsyaml", "type": 301 },
       { "source": "/go/data-driven-fixes", "destination": "https://github.com/flutter/flutter/wiki/Data-driven-Fixes", "type": 301 },
+      { "source": "/go/discontinue", "destination": "/tools/pub/publishing#discontinue", "type": 301 },
```

Comment:
Is this already used in tooling somewhere? 

If not, could we use `/go/pub-discontinue` or `/go/package-discontinue` to be a bit more specific?

Reply:
It is not. I will update it.